### ISSUE TITLE
SNMP - Support for Textual-convention.

### ISSUE THREAD

**Issue by framiere** [influxdata/telegraf#1151]
Hi, 

 Right now `include_instances` is supported which is great.
 However some metrics are expressed using [textual-convention](https://tools.ietf.org/html/rfc1443).
 Here is an example of such a convention
 
``` 
FgHaLBSchedule ::= TEXTUAL-CONVENTION
    STATUS      current
    DESCRIPTION 
        "enumerated type for HA load-balancing schedules"
    SYNTAX      INTEGER { none(1), hub(2), leastConnections(3), 
                    roundRobin(4), weightedRoundRobin(5), random(6), 
                    ipBased(7), ipPortBased(8) }
``` 

 I am thinking about adding support for this feature.
 Any idea/recommandation on how the configuration should look like ?
 
 Cheers

**Comment by sparrc** [comment]
Sounds great, I don't have any opinions on how it should look because I'm not very advanced SNMP user, but if you could post some examples of a few different options you're thinking of, that would be helpful.

**Comment by phemmer** [comment]
Just fyi, with #1389 merged, the new snmp plugin supports conversion of a few different formats (see "conversion" parameter of https://github.com/influxdata/telegraf/tree/master/plugins/inputs/snmp#field-parameters).

It doesn't support all textual convention formats, as full textual convention support would be insanely complex. So I just added the most common use cases. It would be easy to add more if your use case isn't covered.

**Comment by sparrc** [comment]
this is now available in 1.0 RC 1 with the new SNMP plugin, please see here to download the latest version: https://github.com/influxdata/telegraf#linux-deb-and-rpm-packages, and [the changelog](https://github.com/influxdata/telegraf/blob/master/CHANGELOG.md) for details on the new SNMP plugin.
